backend/corpora/dataset_submissions/app.py

```diff
@@ -6,14 +6,18 @@
 
 from sqlalchemy.orm import Session
 
-from backend.corpora.common.entities import Collection
+from backend.corpora.common.entities import Dataset
 from backend.corpora.common.upload import upload
 from backend.corpora.common.utils.db_session import db_session_manager
 from backend.corpora.common.utils.exceptions import CorporaException
 
 logger = logging.getLogger(__name__)
-USERNAME_REGEX = r"[\w\-\|]+"
+USERNAME_REGEX = r"(?P<username>[\w\-\|]+)"
```

Comment:
cooool 🧊 

I did not know about 'named capture groups' in python. Glad to be introduced. 🧠